dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletAPIInterceptor.java

```diff
@@ -764,6 +765,21 @@ public Contentlet findContentletForLanguage(long languageId, Identifier contentl
 		return c;
 	}
 
+	public List<Contentlet> findPagesByTemplate(Template template, User user, boolean respectFrontendRoles) throws  DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 153 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)